modules/ROOT/pages/hdfs/hdfs-connector-studio.adoc

```diff
@@ -0,0 +1,121 @@
+= Use Studio to Configure Hadoop (HDFS) Connector - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To configure a connector in Studio:
+
+. Create a Mule project.
+. Supply your authentication credentials to Anypoint Platform.
+. Add the connector to your Mule project
+. Configure an input source for the connector's flow.
+. Configure the connector.
+
+Follow these best practices when you configure an input source and connector in Studio:
+
+* Configure an associated global element for the input source and connector.
+* Create a YAML or properties file to contain your properties, and then reference the file from the configuration.
+* Configure a reconnection strategy for the input source and target connector.
+
+== Add the Connector to Your Project
+
+Add Hadoop (HDFS) Connector to a Mule project to automatically populate the XML code with the connector's namespace and schema location, as well as add the required dependencies to the project's `pom.xml` file.
+
+. In Studio, create a Mule project.
+. In the Mule Palette view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type HDFS in the search field.
+. Click the Hadoop (HDFS) connector in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+When you design your application in Studio, the act of dragging the connector's operation from the palette onto the Anypoint Studio canvas automatically populates the XML code with the connector namespace and schema location.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met. Possible input source operations for this connector are:
+
+* Read +
+Start your app when the content of a file is read and streamed to the rest of the flow.
+* HTTP Listener +
+Start your app using a browser or a command such as `curl`.
+* Scheduler +
+Start your app at timed intervals.
+
+In Studio, drag the input operation you choose from the *Mule Palette* to the *Source* panel in the canvas.
+
+The required fields for the `Read` input source are:
+
+* Configuration +
+The name of the connector configuration to use
+* Path +
+The path for the file to read 
+
+[[configure_global_element]]
+== Configure a Global Element for the Connector
+
+To configure a connector in your Mule app, first configure a global element that can be used by all instances of that connector in the flow:
```

Comment:
"To configure a connector in your Mule app, first configure a global element" 
The way that this is worded, we have not yet begun to configure the connector (the goal of the topic set), and THIS topic is prerequisite to that.
Again, I feel like I have been reading about how to prepare, in Studio, to configure a connector, not the actual configuration of the connector.